core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -405,87 +153,17 @@ public final ResponseEntity<Object> handleImageUrlParseException(ImageUrlParseEx
-        log.trace(exception.getMessage(), exception);
-        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
-    }
-
-    /**
-     * Method interceptor exception {@link OrderViolationException}.
-     *
-     * @param exception Exception which should be intercepted.
-     * @param request   contain detail about occur exception.
-     * @return ResponseEntity which contain http status and body with message of
-     *         exception.
-     */
-    @ExceptionHandler({OrderViolationException.class})
-    public final ResponseEntity<Object> handleAlreadyExistingAddress(OrderViolationException exception,
-        WebRequest request) {
-        ExceptionResponce exceptionResponse = new ExceptionResponce(getErrorAttributes(request));
-        log.trace(exception.getMessage(), exception);
-        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
-    }
-
-    /**
-     * Method interceptor exception {@link EventsNotFoundException}.
-     *
-     * @param ex         Exception which should be intercepted.
-     * @param webRequest contain detail about occur exception.
-     * @return ResponseEntity which contain http status and body with message of
-     *         exception.
-     */
-    @ExceptionHandler({EventsNotFoundException.class})
-    public final ResponseEntity<Object> handleEventsNotFoundException(EventsNotFoundException ex,
-        WebRequest webRequest) {
-        ExceptionResponce exceptionResponce = new ExceptionResponce(getErrorAttributes(webRequest));
-        log.trace(ex.getMessage(), ex);
-        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponce);
-    }
-
-    /**
-     * Method interceptor exception {@link NotificationNotFoundException}.
-     *
-     * @param exception Exception which should be intercepted.
-     * @param request   contain detail about occur exception.
-     * @return ResponseEntity which contain http status and body with message of
-     *         exception.
-     */
-    @ExceptionHandler({NotificationNotFoundException.class})
-    public final ResponseEntity<Object> handleNotificationNotFoundException(NotificationNotFoundException exception,
-        WebRequest request) {
+    @ExceptionHandler({PaymentNotFoundException.class, PhoneNumberParseException.class,
+        AddressAlreadyExistException.class, OrderViolationException.class,
+        NotificationNotFoundException.class, CertificateNotFoundException.class,
+        ActiveOrdersNotFoundException.class, NotFoundException.class,
+        UnexistingUuidExeption.class, UnexistingOrderException.class,
+        LanguageNotFoundException.class, EventsNotFoundException.class,
+        ReceivingStationNotFoundException.class, PositionNotFoundException.class,
+        EmployeeNotFoundException.class, OrderNotFoundException.class,
+        NotFoundOrderAddressException.class, BagNotFoundException.class,
+        LocationNotFoundException.class, ServiceNotFoundException.class})
```

Comment:
These all exception has a similar status, could we have to create one universal exception e.g. BadRequestException and handle it here?